TileFormats/Instanced3DModel/README.md

```diff
@@ -98,19 +121,51 @@ In either case, `header.gltfByteLength` contains the length of the glTF field in
 
 The `instances` field immediately follows the `glTF` field (which may be omitted when `header.gltfByteLength` is `0`).
 
-The `instances` field contains `header.instancesLength` of tightly packed instances.  Each instance has three fields:
+The `instances` field contains `header.instancesLength` of each instance field which are stored in groups of each field.
```

Comment:
"are stored in groups of each field" -> this needs more explaining, perhaps a small example with a diagram would help, e.g. show `X0X1X2 Y0Y1Y2 Z0Z1Z2` maps to
* instance: X0, Y0, Z0
* instance: X1, Y1, Z1
* instance: X2, Y2, Z2